.github/workflows/test.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '0 */12 * * *'
```

Comment:
What does the schedule do?

Reply:
That schedule is the same as ovn-kubernetes upstream and from openshift-sdn. chron(8) I think it is twice a day, every 12 hours.

Reply:
I think OVN-Kubernetes has a periodic job setup, we probably don't need that. I would just drop and on run job on PR push.

Reply:
I would recommend leaving the schedule in. It is in the doesn't do any harm and might do some good category. It won't squawk unless it fails.